internal/servers/controller/handlers/accounts/account_service.go

```diff
@@ -152,13 +175,24 @@ func (s Service) GetAccount(ctx context.Context, req *pbs.GetAccountRequest) (*p
 	if authResults.Error != nil {
 		return nil, authResults.Error
 	}
-	u, err := s.getFromRepo(ctx, req.GetId())
+	acct, err := s.getFromRepo(ctx, req.GetId())
+	if err != nil {
+		return nil, err
+	}
+
+	outputFields := requests.OutputFields(ctx)
+	item, err := toProto(ctx, acct, handlers.WithOutputFields(&outputFields))
 	if err != nil {
 		return nil, err
 	}
-	u.Scope = authResults.Scope
-	u.AuthorizedActions = authResults.FetchActionSetForId(ctx, u.Id, IdActions[auth.SubtypeFromId(u.Id)]).Strings()
-	return &pbs.GetAccountResponse{Item: u}, nil
+	if outputFields.Has(globals.ScopeField) {
```

Comment:
It would be nice to have all the outgoing proto building logic in one place instead of partially inside toProto and partially inside the caller.  Consider passing the values for these fields into the toProto (maybe using options, or maybe some other way).

Reply:
We were already doing this (toProto was just being called in a helper function) so it's not a new paradigm. I agree that it could be made better, but one reason I didn't do this now is I'm not sure `handlers` is the right place for options here and so I didn't want to unilaterally expand options there without getting some feedback. Once we figure out where it should all live it's easy enough to sweep through and update all of this.

Reply:
I'll address this in the next PR.